src/reducers/todo.jsx

```diff
@@ -49,9 +49,15 @@ export default function todoReducer(state = initialState, action) {
         },
       };
     }
-    // case 'DELETE_TODO': {
-    //   return state.filter((todo) => todo.id !== action.payload);
-    // }
+    case 'DELETE_TODO': {
+      const { [action.payload]: taskWillBeDelete, ...tasks } = state.tasks;
```

Comment:
Please make this line clear: 

What action.payload data refer to ?

if it is ID, please order your data payload like `payload: {id}` and use like
```suggestion
      const { [action.payload.id]: taskWillBeDelete, ...tasks } = state.tasks;
```

Reply:
I get it, and i'll make it clean.